### ISSUE TITLE
Iron 0.5 support

### ISSUE THREAD

**Issue by lfairy** [lfairy/maud#70]
I've made the changes locally -- just need to test and publish to crates.io.

**Comment by lfairy** [comment]
This has been fixed in master, but I want to wait for iron/staticfile#91 to land before cutting a new release.

**Comment by lfairy** [comment]
staticfile has been updated, but the changes have not yet been released. If you really want to use Iron 0.5, then you can make it work using git dependencies:

```toml
[dependencies]
# ...
iron = "0.5.1"
maud = { git = "https://github.com/lfairy/maud.git", features = ["iron"] }
maud_macros = { git = "https://github.com/lfairy/maud.git" }
staticfile = { git = "https://github.com/iron/staticfile.git", features = ["cache"] }
# ...
```

**Comment by ernestas-poskus** [comment]
not sure whether just me or what but getting this when running cargo test

```bash
$ ct
   Compiling maud_macros v0.15.0
error[E0063]: missing fields `close_span`, `open_span` in initializer of `syntax::tokenstream::Delimited`
   --> /home/ow/.cargo/registry/src/github.com-1ecc6299db9ec823/maud_macros-0.15.0/src/parse.rs:418:52
    |
418 |         body.push(TokenTree::Delimited(sp, Rc::new(Delimited {
    |                                                    ^^^^^^^^^ missing `close_span`, `open_span`

error: aborting due to previous error

error: Could not compile `maud_macros`.

To learn more, run the command again with --verbose.
```

Found this commit 10+ days old https://github.com/rust-lang/rust/commit/49f5b0a8cf1a2d588a55f6cb8ea43942e147c66b where delimiters were removed but the odd thing going to definition of 'Delimiter' missing fields are there.

**Comment by lfairy** [comment]
@ernestas-poskus you need to update your compiler -- see #71

**Comment by ernestas-poskus** [comment]
thanks dropped rsvm and started using rustup

**Comment by lfairy** [comment]
Released as [0.16.0](https://crates.io/crates/maud/0.16.0).
